File: BukkitUtil-core/src/main/java/no/kh498/util/jackson/mixin/VectorMixIn.java
package no.kh498.util.jackson.mixin;

import com.fasterxml.jackson.annotation.JsonIgnore;

/**
 * @author Elg
 */
public interface VectorMixIn {

    @JsonIgnore
    int getBlockX();

    @JsonIgnore
    int getBlockY();

    @JsonIgnore
    int getBlockZ();
}

File: BukkitUtil-core/src/main/kotlin/no/kh498/util/jackson/mixin/VectorMixIn.kt
package no.kh498.util.jackson.mixin

import com.fasterxml.jackson.annotation.JsonIgnore

/**
 * @author Elg
 */
interface VectorMixIn {
    @get:JsonIgnore
    val blockX: Int

    @get:JsonIgnore
    val blockY: Int

    @get:JsonIgnore
    val blockZ: Int
}
